src/api/app.py

```diff
@@ -205,11 +205,19 @@ async def uploadHandler(
 #
 #     return user_controller.get_user_info(session_id)
 #
-#
-# @app.route('/api/user', methods=['POST'])
-# def add_user():
-#     return user_controller.add_user(request.json)
-#
+
```

Comment:
For the sake of letting the owner of this endpoint complete it themselves, undo these changes

Reply:
I don't see it locally, but I pulled from FastAPIConversion branch again and merged.
Let me know if it's good to resolve.